src/protocol_edge.c

```diff
@@ -143,8 +140,10 @@ bool add_edge_h(meshlink_handle_t *mesh, connection_t *c, const char *request) {
 
 	if(!from) {
 		from = new_node();
+		from->status.dirty = true;
```

Comment:
In many places node `status.dirty` is set to true are we anywhere in the code writing out the configuration of the node and resetting node's dirty status? 
If not so maybe it can be synchronized when meshlink thread has some idle time and if thread is busy with mesh event handling then do it later when it finds idle time.

Reply:
Hm, that code seems to have gotten lost when I rebased it. It should write it out when the periodic or pingtimeout handler runs. Also, since you now have tests that do `raise(SIGINT)`, I'll check that any updates to node config files that you would expect to have been stored immediately are indeed written out immediately.